media-gfx/yafaray/yafaray-3.4.1-r1.ebuild

```diff
@@ -0,0 +1,114 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8} )
```

Comment:
You can edit `PYTHON_COMPAT` without revbumping. I'd rather you do that if there's nothing else happening here.

I can already see this confusing portage a lot, since -r0 has py-3.6 and -r1 has py-3.7 & py-3.8.

Reply:
Ok, renamed it back.